test/unit/transpiler/passes/scheduling/control_flow_test_case.py

```diff
@@ -15,8 +15,8 @@
 from typing import Any, Optional
 
 from qiskit import QuantumCircuit
-from qiskit.test._canonical import canonicalize_control_flow
 
+from .....utils import canonicalize_control_flow
```

Comment:
We can actually remove this file, it was removed in `main` but accidentally added again when we updated `experimental-0.2`